src/components/bcros/Alert.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div data-cy="alert-display" class="px-3 py-3">
+    <h3>
+      <UIcon
+        :class="`${iconColour} mr-2`"
+        :name="iconName"
+        data-cy="contact-icon"
+      />
+      <span class="font-semibold flex-auto">{{$t(alertHeader)}}</span>
+      <button class="float-right text-blue-600 hover:bg-blue-100 text-sm px-1 py-1mr-4" href="#" @click="expanded = !expanded">
```

Comment:
can you replace 'text-blue' with 'text-primary'? It will evaluate the same right now, but if we ever change our primary colour in the future we would want this to change with it. Also, did 600 come from the UX or should that be '500'? I think we are using '500' everywhere else for these sort of things

Reply:
No particular reason for the 600 will change

Reply:
Fixed